dune-project

```diff
@@ -1,4 +1,4 @@
-(lang dune 3.8)
+(lang dune 3.9)
```

Comment:
This change is clearly requiring 3.9

Reply:
Oh, thought you talked about Reason, not dune.

Reply:
But melange only requires dune 3.8

Reply:
The current state of this repo is very broken, since it requires melange master and Reason 3.9.0 for the snapshot testing + the reason-react-ppx from the repo

So there should be a few things to un-tangle before we can even publish newer versions